.github/pull_request_template.md

```diff
@@ -14,7 +14,8 @@
 * [ ] All new Python files are placed inside an existing directory.
 * [ ] All filenames are in all lowercase characters with no spaces or dashes.
 * [ ] All functions and variable names follow Python naming conventions.
-* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
+* [ ] All function parameters and return values are annotated with Python [mypy](https://docs.python.org/3/library/typing.html).
```

Comment:
Please undo.  These are type hints.  Mypy and Pyright are both programs that test type hints.